tests/jballerina-unit-test/src/test/resources/test-src/expressions/access/member_access.bal

```diff
@@ -1178,11 +1178,12 @@ function testUnspecifiedFieldRecordMemberAccessWithStringCharKeyExpr() {
     assertTrue(v is ());
 }
 
-type INT int;
-type INT2 int|int:Signed16;
-type STR2 string|string:Char;
+type IntType int;
+type IntType2 int|int:Signed16;
+type Str2 string|string:Char;
+type IntUnion 1|2|3|4;
```

Comment:
We better use a consistent pattern when naming. It improves readability.

IntType, Str2 is not consistent. 
IntUnion is again not good as `int|int:Signed16` is also an int union.  

HBT IntType, IntType2, StrType, IntType3?

Reply:
Instead of `IntType` better to use `IntType1`

Reply:
+1